foundry.toml

```diff
@@ -3,7 +3,7 @@ src = 'contracts'                                             # the source direc
 test = 'test'                                                 # the test directory
 out = 'artifacts'                                             # the output directory (for artifacts)
 cache = true                                                  # whether to cache builds or not
-force = true                                                  # whether to ignore the cache (clean build)
+force = false                                                 # whether to ignore the cache (clean build)
```

Comment:
Foundry would recompile contracts/tests if they are modified so I think we can disable force build to save tens of seconds each time running tests locally